src/components/Form/From/index.tsx

```diff
@@ -1,89 +1,52 @@
-import { Fragment } from 'preact';
-import { Input } from 'react-daisyui';
 import { UseFormRegisterReturn } from 'react-hook-form';
+import { StateUpdater } from 'preact/hooks';
 import { Asset } from 'stellar-sdk';
-import AssetSelector from '../../Selector/AssetSelector';
+
+import { BlockchainAsset } from '../../Selector/AssetSelector/helpers';
+
+import { SwapFrom } from './variants/SwapFrom';
+import { StandardFrom } from './variants/StandardFrom';
 
 export interface FromProps {
   className?: string;
   max?: number;
   min?: number;
   register: UseFormRegisterReturn;
-  setValue: (n: number) => void;
-  assets?: Asset[];
-  selectedAsset?: Asset;
-  setSelectedAsset?: (a: Asset) => void;
+  setValue?: (n: number) => void;
+  assets?: BlockchainAsset[];
+  selectedAsset?: BlockchainAsset;
+  setSelectedAsset?: StateUpdater<BlockchainAsset | undefined> | StateUpdater<Asset | undefined>;
   network?: string;
   assetSuffix?: string;
   error?: string;
+  customText?: string;
+  minBadge?: {
+    value: string;
+    onClick?: () => void;
+  };
```

Comment:
Nit: we could extract those to something like `BadgeProps` and re-use the same for min and max badge.